my_project/tab_summary/app_summary.py

```diff
@@ -89,7 +91,7 @@ def layout_summary():
                         [
                             dbc.Col(
                                 html.Label(
-                                    "Heating degree day (HDD) setpoint (°C)",
+                                    "Heating degree day (HDD) setpoint (default: °C) / °F",
```

Comment:
This should be updated as a function of the user selection

Reply:
As you have done below in line 224

Reply:
This problem is now solved. The set number will be changed automatically according to the si-ip-state.

Reply:
please push the changes since you are mentioning that the problem is solved but I cannot see the new code.